custom/icds_reports/utils/aggregation_helpers/ls_awc_visit_form.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from dateutil.relativedelta import relativedelta
+from custom.icds_reports.const import AGG_LS_AWC_VISIT_TABLE
+from custom.icds_reports.utils.aggregation_helpers import BaseICDSAggregationHelper, month_formatter
+import hashlib
+
+class LSAwcMgtFormAggHelper(BaseICDSAggregationHelper):
+    ucr_data_source_id = 'static-awc_mgt_forms'
+    aggregate_parent_table = AGG_LS_AWC_VISIT_TABLE
+    aggregate_child_table_prefix = 'icds_db_ls_awc_mgt_form_'
+
+    def __init__(self, month):
+        self.month = month
+
+    def drop_table_if_exists(self, agg_level):
+        return """
+        DROP TABLE IF EXISTS "{table_name}"
+        """.format(table_name=self.self.generate_child_tablename(self.month))
+
+    def create_table_query(self, month=None):
+        month = month or self.month
+        month_string = month_formatter(month)
+        tablename = self.generate_child_tablename(month)
+
+        return """
+        CREATE TABLE IF NOT EXISTS "{child_tablename}" (
+            CHECK (month = %(month_string)s,
+            LIKE "{parent_tablename}" INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES
+        ) INHERITS ("{parent_tablename}")
+        """.format(
+            parent_tablename=self.aggregate_parent_table,
+            child_tablename=tablename,
+        ), {
+            "month_string": month_string
+        }
+
+    def generate_child_tablename(self, month=None):
```

Comment:
same thing as the create query, this is the same as the base class except not partitioned by state. i think it would be cleaner if you just partitioned by state, and it comes with the added scalability benefits